### PR TITLE
[#1684] fix(web): fix special characters displays

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

image

### Why are the changes needed?

Fix: #1684 

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

File: web/app/ui/DetailsDrawer.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ const DetailsDrawer = props => {
       const propsData = Object.keys(drawerData.properties).map(item => {
         return {
           key: item,
-          value: drawerData.properties[item]
+          value: JSON.stringify(drawerData.properties[item]).replace(/^"|"$/g, '')
```
Comment (jerryshao) [inline web/app/ui/DetailsDrawer.js:56]: What's the meaning of `replace(/^"|"$/g, '')`?

Comment (ch3yne) [reply web/app/ui/DetailsDrawer.js:56]: The JSON.stringify method will return a string wrapped in double quotes.

```
JSON.stringify('abc')
# "abc"
```

**Review by jerryshao — DISMISSED** [review]

**Comment by jerryshao** [comment]
@ch3yne please rebase this code.

**Comment by ch3yne** [comment]
> @ch3yne please rebase this code.

DONE

**Review by jerryshao — APPROVED** [review]

**Pull request merged**